packages/examples/src/relay/pure-relay/Feedback.stories.tsx

```diff
@@ -173,11 +176,19 @@ export const LikeFailure: Story = {
         operation,
         {
           Feedback: () => sampleFeedback,
+          ViewData: () => ({
+            viewDataField: "View data field",
+            likeLabel: "Like",
+            unlikeLabel: "Unlike",
+            numberOfLikesLabel: "You have {count} likes",
+          }),
```

Comment:
Nit: I think it would speak more to the imagination if the example was using different languages for the string definition vs the runtime value [for the current locale]. Not a biggie, though.